datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -102,6 +108,27 @@ const SearchBarContainer = styled.div`
     text-align: center;
 `;
 
+const ExploreAllButtonContainer = styled.div`
+    display: flex;
+    justify-content: right;
+`;
+
+const ExploreAllButton = styled(Button)`
+    && {
+        padding: 0px;
+        margin: 0px;
+        height: 16px;
+    }
```

Comment:
just in general I'm confused why we have these `&&` or `&&&` all over our css styling... do you understand it?

at least to my understanding, using `&` is meant to apply styles to the current selector you're on as well as above selectors, so idk if it's really doing anything in these situations, but I could be wrong since I've never seen it used other than with `&:hover { ...` or some pseudo selector

also I recognize you may be copy pasting this styling from elsewhere lol

Reply:
no so for some reason i have to do this to bypass the native antd styles in some cases. would love to understand why

Reply:
fricken ant design man